scripts/fast-reboot

```diff
@@ -3,6 +3,7 @@
 REBOOT_USER=$(logname)
 REBOOT_TIME=$(date)
 REBOOT_CAUSE_FILE="/host/reboot-cause/reboot-cause.txt"
+PLATFORM_REBOOT_PRE_CHECK="platform_reboot_pre_check"
```

Comment:
Minor: suggest moving this down to where other `PLATFORM` variables are defined.

Reply:
Updated. Thank you very much.